packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,42 @@
+import React, { Children } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { Box } from '../../Box';
+import { Crumb } from './Crumb';
+import { CrumbLink } from './CrumbLink';
+import { CrumbSimpleMenu } from './CrumbSimpleMenu';
+import { Divider } from './Divider';
+import { Flex } from '../../Flex';
+
+const ListFlex = styled.ol`
+  display: flex;
+`;
+
+export const Breadcrumbs = ({ label, children, ...props }) => {
+  return (
+    <Box aria-label={label} {...props}>
+      <ListFlex aria-hidden={true}>
+        {Children.map(children, (child, index) => {
+          const isLast = index + 1 === children.length;
+
+          return (
+            <Flex inline as="li">
```

Comment:
```
const Li = styled(Flex)`
  span,
  a {
    color: ${({ theme, isLast }) => (isLast ? theme.colors.neutral800 : theme.colors.neutral600)};
    font-weight: ${({ theme, isLast }) => (isLast ? theme.fontWeights.bold : theme.fontWeights.regular)};
  }
`;

...

<Li isLast={isLast} inline as="li">
```

then we can remove the styling from Crumb/CrumbLink?

Reply:
I don't want to do this, for 2 reasons:

- it would make the last `CrumbLink` bold, which is not a behavior we want (the last `Crumb` should never be a link)
- it introduces a `span` element selector, which is brittle and might affect other elements
- it moves styling logic which belongs into the `Crumb` to the parent